.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish Builds
+
+on:
+  release:
+    types: [published]
```

Comment:
Still not sure whether it's better to trigger on tag or release - perhaps a tag is easier to create?